README.md

```diff
@@ -32,7 +33,6 @@ bareos_clients:
     password: MySuperSecretPassword
     enable_backup_job: true
     state: present                      # optional
-    autostart: true                     # optional
```

Comment:
`run job=backup-{{ item.name }} client={{ item.name }} yes | bconsole` forces a first backup on a freshly registered client, no matter the job name (the jobs aren't always named `backup-*`), and fails if none corresponds. We can't easily get the job name since we are looping through clients and not jobs, and I'd prefer we run the first job interactively to check everything's fine.

Reply:
I just don't see the point to remove features which we can already turn on or off with a tunable. We can still discuss changing the default of the tunable.

Reply:
I added it back, but with the default set to `false`

Reply:
That is a breaking change you need to document for the next release.